tests/test_s3_specific.py

```diff
@@ -82,7 +86,7 @@ def _download_with_threads(s3_rig, tmp_path, use_threads):
         assert not p.exists()
 
     finally:
-        p = s3_rig.create_cloud_path("/dir_0/file0_0.txt")
+        p = s3_rig.create_cloud_path(f"{s3_rig.test_dir}/dir_0/file0_0.txt")
```

Comment:
I'm not sure what the purpose of this `finally` block is for this test, but again, same comment that `s3_rig.create_cloud_path` already handles `test_dir`.

Reply:
Looks like cleanup since in order to pass the TransferConfig we have a file created outside of the test rig:
https://github.com/drivendataorg/cloudpathlib/blob/7f6943265ae6fc98a2595b28cb98d996672fd209/tests/test_s3_specific.py#L70-L71

Reply:
That's not even the same filename though. `file0_0.txt` vs. `file0_to_download.txt`.

Reply:
🤔 good point. I'll try pulling it out, but it's not really part of this change so will file an issue if something breaks

Reply:
Ran twice locally and watched the actual S3 bucket to confirm it was cleaned up after the test ran. I think should be good to go now.